std/digest/murmurhash.d

```diff
@@ -200,15 +200,23 @@ struct MurmurHash3(uint size /* 32 or 128 */ , uint opt = size_t.sizeof == 8 ? 6
 
         alias Element = uint[4]; /// The element type for 128-bit implementation.
 
-        this(uint seed4, uint seed3, uint seed2, uint seed1)
+        this(Element seed) pure nothrow @nogc
+        {
+            h4 = seed[4 - 1];
+            h3 = seed[3 - 1];
+            h2 = seed[2 - 1];
+            h1 = seed[1 - 1];
+        }
```

Comment:
ok, i will just remove them. If some one wants them, then he can open PR and write unittests :-)